dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java

```diff
@@ -332,4 +334,35 @@ private static NetworkInterface findOuterAddress(List<NetworkInterface> validNet
         return networkInterface;
     }
 
+    /**
+     * check if address is legal address, a legal address should be ip:port
+     * @param ipAndPortAddress address to check
+     * @return true if address is legal
+     */
+    public static boolean isLegalAddress(String ipAndPortAddress) {
+        if (StringUtils.isEmpty(ipAndPortAddress)) {
+            return false;
+        }
+        String[] ipAndPort = ipAndPortAddress.split(":");
+        if (ipAndPort.length != 2) {
+            return false;
+        }
+        return isValidIPv4Address(ipAndPort[0]) && isValidPort(ipAndPort[1]);
```

Comment:
Can this work on ipv6?

Reply:
> Can this work on ipv6?

now cannot, i will add ipv6

Reply:
i add ipv6 validator, refer to [RFC 3896](https://www.ietf.org/rfc/rfc3986.txt)